### DISCUSSION TITLE
Low/No code platform for better deployment and Training of CV models

### DISCUSSION THREAD

**Discussion by Synlabs01** [community/community#35318] [Other Feature Feedback, Questions, & Ideas]
Synergylabs has fast become a dynamic and fast-moving company in Client Solution Management using emerging in Technology in India and has proven itself to be one of the market leaders in INDUSTRY 4.0, FINTECH AND ARTIFICIAL INTELLIGENCE.

Established in 2017 by Ex top Goldman Sachs, IBM executives, we have helped several marquee customers in leveraging technology and achieve cutting edge over the competitors such as Indian Oil, Euro Car Parks, Flipkart, Navyug Cycles to name a few.

We strive to provide you with the best solutions to your business system needs. As our client, we place great emphasis and focus on your problems and goals and develop solutions that best fits your needs. We offer strong and effective solutions to your business with an enduring impact. We think of ourselves as your long-term partner.

**We are building a low code/no code platform to train and ship computer vision models quickly with plug and play tools to connect to your favorite annotation platform. To get an early access and free inference credits kindly visit** 
[shipflo.ai](https://shipflo.ai/)

**Comment by janggohebat07** [comment]
Pada tanggal Kam, 6 Okt 2022 18.00, Synlabs01 ***@***.***>
menulis:
